frodo.js

```diff
@@ -2,88 +2,94 @@
  * Extras Plugin
```

Comment:
SCRIPT-There seems to be a typo in the function name. Did you mean `calculateAverage` instead of `calculateAverge`?